test/functionalTest/cases/000_ngsi10_convenience/individual_context_entity_attributes_get_fails.test

```diff
@@ -44,7 +44,6 @@ Date: REGEX(.*)
     <entityId type="" isPattern="false">
       <id>entity01</id>
     </entityId>
-    <contextAttributeList/>
```

Comment:
Yeah, this was another improvement we got for free.
Different render method - my guess.
I haven't really spent any time in trying to understand these changes, as long as I've seen them as OK.